strawberryfields/ops.py

```diff
@@ -792,17 +792,18 @@ class Catstate(Preparation):
     A cat state is the coherent superposition of two coherent states,
 
     .. math::
-       \ket{\text{cat}(\alpha)} = \frac{1}{N} (\ket{\alpha} +e^{i\phi} \ket{-\alpha}),
+       \ket{\text{cat}(\alpha)} = \frac{1}{N} (\ket{\alpha} +e^{i\theta} \ket{-\alpha}),
 
-    where :math:`N = \sqrt{2 (1+\cos(\phi)e^{-2|\alpha|^2})}` is the normalization factor.
+    where :math:`N = \sqrt{2 (1+\cos(\theta)e^{-2|\alpha|^2})}` is the normalization factor and :math:`\alpha = a e^{i\phi}`.
```

Comment:
Turned to use the polar form of `\alpha = a e^{i\phi}` which necessitated the renaming of `phi` to `theta`. Hence the new appearances of `theta`.